### ISSUE TITLE
Gemini has returned a different issue

### ISSUE THREAD

**Issue by CevreMuhendisi** [MaKTaiL/gemini-srt-translator#3]
"Gemini has returned a different number of lines than the original , trying again" 
I'm getting this error, is there a solution for it?

![image](https://github.com/user-attachments/assets/93feeefc-15ab-4148-80c8-11e2b4e3da6f)

**Comment by MaKTaiL** [comment]
What version are you using? This should be fixed in v1.1.4

**Comment by MaKTaiL** [comment]
Hi, seems to be a problem with how Gemini sometimes handles some dialogs (it decides to arbitrarily split some dialogs in two). I got a fix coming up in the next version with a workaround for this. Hopefully it will solve the problem for good.

**Comment by ongnxco** [comment]
I have the same problem and after updating gemini-srt-translator-1.2.0 a new error appeared. 

```
Traceback (most recent call last):
  File "~/gemini-trans/main.py", line 1, in <module>
    import gemini_srt_translator as gst
  File "~/gemini-trans/.venv/lib/python3.11/site-packages/gemini_srt_translator/__init__.py", line 32, in <module>
    from .main import GeminiSRTTranslator
  File "~/gemini-trans/.venv/lib/python3.11/site-packages/gemini_srt_translator/main.py", line 159
    translated_subtitle[int(line["index"])].content = f"\u202B{line["content"]}\u202C"
                                                                     ^^^^^^^
SyntaxError: f-string: unmatched '['
```

**Comment by MaKTaiL** [comment]
@ongnxco this should be fixed now in version v1.2.1, sorry about that. ;)
